src/main/java/com/github/kuros/random/jpa/resolver/CreationOrderResolverImpl.java

```diff
@@ -7,14 +7,7 @@
 import com.github.kuros.random.jpa.types.Entity;
 import com.github.kuros.random.jpa.types.EntityHelper;
 
-import java.util.Collection;
```

Comment:
Please do not collapse imports.

Reply:
Ok, I add them back, it comes from IDE format plugin